### PR TITLE
Adding git quickstart section to README

### PR DESCRIPTION
**Pull request by igor-ribeiiro**
Just updating the README to have a quick and easy introduction to start out contributing to this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one, and let us know your GitHub username. We need to add you to the team in GitHub.
```
Comment (akrantz) [inline README.md:107]: The part ", and let us know..." can be removed as it doesn't apply.

File: README.md [hunk 2]
```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
...
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
+4. Create a fork off of OfficeDev/Office-Addin-Scripts in GitHub
+5. In a command prompt:
+    1. git remote add :username: :fork-repo-url:     (e.g. git remote add :username: https://github.com/:username:/Office-Addin-Scripts.git --- where “:username:” is your GitHub user name)
```
Comment (akrantz) [inline README.md:111]: I think putting the parameters inside curly braces ( {username} ) or back-ticks ( `username` ) may work better than using colons.

File: README.md [hunk 3]
```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
...
+    2. git checkout -b my-branch-name (recommend using all lowercase with hyphens for branch names)
+6. Make your changes to remove debug method logging (I mention where the change should be made in the task I assigned to you)
```
Comment (akrantz) [inline README.md:113]: This step is a little too specific.  :-) "Make your desired changes."

**Review by akrantz — CHANGES_REQUESTED** [review]
See comments.

**Comment by igor-ribeiiro** [comment]
Just fixed the comments leaved by @akrantz.

File: README.md [hunk 4]
```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one.
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
+4. Create a fork off of OfficeDev/Office-Addin-Scripts in GitHub
+5. In a command prompt:
```
Comment (Rick-Kirkham) [inline README.md:110]: Should specify what folder to CD to before running these commands.

**Review by Rick-Kirkham — APPROVED** [review]
approving with a suggestion.

File: README.md (hunk 4 above)
Comment (akrantz) [reply README.md:110]: @igor-ribeiiro Ricky makes a good point. For step 5 (in a command prompt), please add a first step: `1. cd Office-Addin-Scripts`

Also, a step 8 could be added. 8. Go to https://github.com/OfficeDev/Office-Addin-Scripts and create a pull request.

File: README.md [hunk 5]
```diff
@@ -101,6 +101,22 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one.
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
```
Comment (akrantz) [inline README.md:108]: We could move the clone into the command prompt steps.

Make step 3. Go to https://github.com/OfficeDev/Office-Addin-Scripts.
Make step 4. Create a fork or the repository for yourself.

Then add a first step under "In a command prompt:" and we can improve the "git remote add" step:
        1. git clone https://github.com/OfficeDev/Office-Addin-Scripts
        2. cd Office-Addin-Scripts
        3. git remote add {username} https://github.com/{username}/Office-Addin_Scripts.git 
            (where "{username}" is your GitHub user name.) 
         4. git checkout -b branch-name
             (recommend using all lowercase with hyphens for branch names)

**Review by akrantz — APPROVED** [review]
Thanks for making the changes. Everything looks good now.

**Pull request merged**